python_modules/dagster-test/dagster_test/graph_job_op_toys/schedules.py

```diff
@@ -57,160 +29,85 @@ def _toys_tz_info():
+        return {}
 
-    return partition_set.create_schedule_definition(
-        schedule_name=schedule_name,
-        cron_schedule="* * * * *",  # tick every minute
-        partition_selector=backfilling_partition_selector,
-        should_execute=_should_execute,
-        execution_timezone=_toys_tz_info(),
+    return schedule_from_partitions(
+        many_events.to_job("many_events_hourly_config", config=hourly_materialization_config),
+        name="many_events_hourly",
     )
 
 
-@hourly_schedule(
-    pipeline_name="many_events",
-    start_date=datetime.datetime(2021, 1, 1),
-    execution_timezone=_toys_tz_info(),
-)
-def hourly_materialization_schedule():
-    return {}
-
-
-@daily_schedule(
-    pipeline_name="many_events",
-    start_date=datetime.datetime(2021, 1, 1),
-    execution_timezone=_toys_tz_info(),
-)
 def daily_materialization_schedule():
-    return {}
+    @daily_partitioned_config(start_date=datetime.datetime(2021, 1, 1), timezone=_toys_tz_info())
+    def daily_materialization_config(_start, _end):
+        return {}
+
+    return schedule_from_partitions(
+        many_events.to_job("many_events_daily_config", config=daily_materialization_config),
+        name="many_events_daily",
+    )
 
 
-@weekly_schedule(
-    pipeline_name="many_events",
-    start_date=datetime.datetime(2021, 1, 1),
-    execution_timezone=_toys_tz_info(),
-)
 def weekly_materialization_schedule():
-    return {}
+    @weekly_partitioned_config(start_date=datetime.datetime(2021, 1, 1), timezone=_toys_tz_info())
+    def weekly_materialization_config(_start, _end):
+        return {}
+
+    return schedule_from_partitions(
+        many_events.to_job("many_events_weekly_config", config=weekly_materialization_config),
+        name="many_events_weekly",
+    )
 
 
-@monthly_schedule(
-    pipeline_name="many_events",
-    start_date=datetime.datetime(2021, 1, 1),
-    execution_timezone=_toys_tz_info(),
-)
 def monthly_materialization_schedule():
-    return {}
+    @monthly_partitioned_config(start_date=datetime.datetime(2021, 1, 1), timezone=_toys_tz_info())
+    def monthly_materialization_config(_start, _end):
+        return {}
+
+    return schedule_from_partitions(
+        many_events.to_job("many_events_monthly_config", config=monthly_materialization_config),
+        name="many_events_monthly",
+    )
 
 
 def longitudinal_schedule():
     from .longitudinal import longitudinal_job
 
-    schedule_name = "longitudinal_demo"
-
-    def longitudinal_config(partition):
+    @daily_partitioned_config(start_date="2020-01-01", timezone=_toys_tz_info())
+    def longitudinal_config(start, _end):
         return {
-            "solids": {
-                solid.name: {"config": {"partition": partition.name}}
-                for solid in longitudinal_job.solids
+            "ops": {
+                op.name: {"config": {"partition": start.to_date_string()}}
+                for op in longitudinal_job.solids
```

Comment:
Need to set access the name of each solid in order to generate config, so still referring to `.solids`. Not sure if there is a better way, or if we just need to change this later on

Reply:
Mind making sure that we're tracking the need to add an `ops` attr?  Might get fixed by @dpeng817 's `JobDefinition` addition?

Reply:
yep, will add to quip